src/renderer/component/video/internal/player.jsx

```diff
@@ -51,7 +56,11 @@ class VideoPlayer extends React.PureComponent {
     // use renderAudio override for mp3
     if (VideoPlayer.MP3_CONTENT_TYPES.indexOf(contentType) > -1) {
       this.renderAudio(container, null, false);
-    } else {
+    }
+    // Render custom viewer: FileRender
+    if (this.fileType()) this.renderFile();
+    // Render default viewer: render-media (video, audio, img, iframe)
+    else if (this.supportedType()) {
```

Comment:
Should this be an `if, else if, else` statement instead?

It looks like it could call `renderAudio` and the default viewer both.